### ISSUE TITLE
Flexbox layout is not laying out properly in Firefox

### ISSUE THREAD

**Issue by swetland** [jtgans/6502ops#1]
1. It would be nice if the info-box received focus when popped up, so keyboard scrolling acted on it rather than the (obscured) instruction list
2. It would be useful to be able to link directly to an instruction (maybe by tacking #opcode on the end of the URL)

**Comment by jtgans** [comment]
Okay, give it a shot now. I made the description full frame and adjusted a whole bunch to make it work using anchors.

**Comment by swetland** [comment]
It's looking good!

Two things:
- after you pop up instruction info and then dismiss it the #opcode remains in the url such that if you reload the page or bookmark it or whatnot, you end up back on the instruction info
- layout is a little weird (see attached) in Firefox on Ubuntu -- it *almost* manages to get all the instructions on one page tidily.  looks similar on Chromium on Ubuntu

![layout](https://user-images.githubusercontent.com/94628/51782346-8bfc3880-20db-11e9-8b94-ac392b48379b.png)

**Comment by jtgans** [comment]
Yeah. I'm not exactly sure what is going on with that. It's like flexbox isn't reacting to the width properly.
